Resources/Prototypes/SoundCollections/broken device.yml

```diff
@@ -0,0 +1,15 @@
+﻿- type: soundCollection
+  id: BrokenDevice
+  files:
+  - /Audio/Items/electrical_short_circuit.ogg
+
+- type: soundCollection
+  id: ShortCircuit
+  files:
+  - /Audio/Items/electrical_short_circuit2.ogg
```

Comment:
Two of your new audio files are in Audio/Effects/ and one is in Audio/Items/
Move last one from items to effects as well and fix the file paths here to get rid of the yaml linter error.
The audio files will also need attributions and a license in `attributions.yml`

Reply:
I hope what I've done will work